README.md

```diff
@@ -41,6 +41,7 @@ You'll also need to add the JDBC driver needed for your database.
 | `:database-name`            | No       | None                   | This property sets the database name.                                                                                                                                                                                                                                                                                                  |
 | `:server-name`              | No       | Depends on the adapter | This property sets the hostname client connects to.                                                                                                                                                                                                                                                                                    |
 | `:port-number`              | No       | Depends on the adapter | This property sets the port clients connects on.                                                                                                                                                                                                                                                                                       |
+| `:connection-test`          | No       | None                   | This property will toggle whether connection test is performed, must be explicitly set to `false` in order to disable the test. |
```

Comment:
Can we assume it's `true` by default? I always prefer to be explicit :)

Reply:
@yogthos WDYT about this one? :)

Reply:
Don't have a strong feeling on this one. :) It's an optional field in hikari config, so it ends up defaulting to true there when not set.